package.json

```diff
@@ -5,12 +5,20 @@
   "author": "Jamon Holmgren <jamon@infinite.red>",
   "version": "0.1.0",
   "homepage": "https://github.com/react-native-community/react-native-webview#readme",
+  "scripts": {
+    "flow": "flow check",
+    "flow-android": "flow check --flowconfig-name .flowconfig.android"
+  },
   "peerDependencies": {
     "react": "^16.0",
-    "react-native": "^0.56"
+    "react-native": "~0.57.0-rc.4"
```

Comment:
0.56 does work with this repo, is there a reason to lock to rc 4?

Reply:
I will see if I can make `flow` pass on 0.56

Reply:
If it's a lot of work, we can require 0.57+.

Reply:
So I discovered why I was having so much trouble with 0.56's JS files, as opposed to 0.57's: 0.56 was tested against Flow 0.75, and [Flow 0.78 introduced](https://github.com/facebook/flow/blob/master/Changelog.md#0780) the `--flowconfig-name` flag which allowed flow to use the `.flowconfig.android` file in the main repo and test the flow files - previously, it seems flow was not ran against android platform files.

0.57, now being tested against Flow 0.80, has a bunch of comments like this in the Android files:

```js
/* $FlowFixMe(>=0.78.0 site=react_native_android_fb) This issue was found when
 * making Flow check .android.js files. */
```

Without them, on 0.56, I get all these errors:

https://gist.github.com/empyrical/70e7cceb87c6e8f89c5f01c120d4d9d3

So I suppose if you want to support 0.56 on this repo, Flow could only be ran against 0.57+ but other tests could be ran on 0.56

Reply:
I'd like to support 0.56 if we can -- I'm totally fine if Flow is not run against 0.56 at all and we just manually test it.

Reply:
I'll bump the version back down to 0.56, and add a note that flow will currently only pass against 0.57